augur/export_v2.py

```diff
@@ -878,7 +883,10 @@ def register_parser(parent_subparsers):
     optional_settings = parser.add_argument_group(
         title="OPTIONAL SETTINGS"
     )
-    optional_settings.add_argument('--minify-json', action="store_true", help="export JSONs without indentation or line returns")
+    minify = optional_settings.add_mutually_exclusive_group()
+    minify.add_argument('--minify-json', action="store_true", help="export JSONs without indentation or line returns, regardless of how many nodes are in the tree.")
```

Comment:
We might want to add in the CLI help (and therefore also in the docs) that there's a default configured somehow - just so that users aren't confused if the output is sometimes minified sometimes not.

Reply:
Added in ba65ef239c5c03a7d21ac68b8a411653953a314a:

https://github.com/nextstrain/augur/blob/ba65ef239c5c03a7d21ac68b8a411653953a314a/augur/export_v2.py#L883-L892

Reply:
Really nice work here -- argument groups (and group help messages) are so helpful for complex commands.